app/[locale]/(layout)/contribute/_components/issues.tsx

```diff
@@ -58,6 +62,32 @@ export const Issues = () => {
     }
   }, [page]);
 
+  const contributeListJsonLd: WithContext<ItemList> = {
+    "@context": "https://schema.org",
+    "@type": "ItemList",
+    name: t("JsonLd.ItemList.name"),
+    itemListElement: (projectIssues?.issues ?? []).map((issue, idx) => ({
+      "@type": "ListItem",
+      position: idx + 1 + ((projectIssues?.pageInfo.currentPage ?? 1) - 1) * 12, // TODO: Change 12 by pageSize when implemente
+      item: {
+        "@type": "CreativeWork",
+        url: issue.url,
+        name: issue.title,
+        dateCreated: issue.createdAt,
+        dateModified: issue.updatedAt,
+        isPartOf: "https://github.com/owner/repo",
```

Comment:
a verif si pas mieux de mettre 
```suggestion
        isPartOf: "https://github.com/DyingStar-game/website",
```